java/gandiva/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set(GANDIVA_PROTO_DIR ${CMAKE_CURRENT_SOURCE_DIR}/proto)
 get_filename_component(GANDIVA_PROTO_FILE_ABSOLUTE ${GANDIVA_PROTO_DIR}/Types.proto
                        ABSOLUTE)
 
+set(Protobuf_USE_STATIC_LIBS ON)
 find_package(Protobuf REQUIRED)
 if(MSVC)
```

Comment:
It's not related to this pull request but could you remove this `if(MSVC)`? It's not needed with recent CMake.

Reply:
Done.